www/html/js/webJS.js

```diff
@@ -46,8 +47,102 @@ $(document).ready(function() {
 
 
     });
+
 });
 $(document).ready(function() {
     $(".kt-tabs-component-content").hide();
     $("#aboutsite").show();
-});
\ No newline at end of file
+
+});
+//tracking navbar
+$(document).ready(function() {
+    $("li").click(function() {
+
+        //set global variables
+        var city;
+        var country;
+        var page = $(this).text();
+
+
+        // current date
+        var currentdate = new Date();
+        var datetime = currentdate.getFullYear() + "-" +
+            (currentdate.getMonth() + 1) + "-" +
+            currentdate.getDate() + " " +
+            currentdate.getHours() + ":" +
+            currentdate.getMinutes() + ":" +
+            currentdate.getSeconds();
+
+        //city and country
+        $.get("http://ipinfo.io", function(response) {
+
+            city = response.city;
+            country = response.country;
+
+            //call function
+            updateNavbarTracking(page, datetime, city, country);
+
+        }, "jsonp");
+
+
+
+
+
+
+    });
+});
+//ajax function
+function updateNavbarTracking(page, time, city, country) {
+
+
+
+    $.ajax({
+        url: 'index.php/landing/trackingnavigation',
+        type: 'post',
+        dataType: 'json',
+        data: { 'page': page, 'time': time, 'city': city, 'country': country }
+    });
+}
+
+
+//tracking landings
+$(document).ready(function() {
+    //set global variables
+    var city;
+    var country;
+
+    // current date
+    var currentdate = new Date();
+    var datetime = currentdate.getFullYear() + "-" +
```

Comment:
You have to insert a datetime format into the database since I requested the column to be of that format. I'm not sure whether a string will properly convert to the desired format. Please test.